src/components/Voting.js

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react";
+import { PatchArticleVotes } from "./api-logic";
+
+const Voting = ({ SingleArticle, article_id }) => {
+  const [vote, setVote] = useState(0);
+  const [appendedVote, setAppendedVote] = useState(SingleArticle.votes);
```

Comment:
Do we need this value here, or could we just use SingleArticle from state already?

Also, this will need to be changed to singleArticle to not confuse it with capitalised components.